angel-ps/core/src/main/java/com/tencent/angel/conf/AngelConfiguration.java

```diff
@@ -28,7 +28,7 @@
 import com.tencent.angel.worker.task.BaseTask;
 
 import org.apache.hadoop.conf.Configuration;
-import org.apache.hadoop.mapreduce.lib.input.CombineTextInputFormat;
+import org.apache.hadoop.mapred.lib.CombineTextInputFormat;
```

Comment:
Thanks for your pr. It is not a good idea to modify the default parameter value. If you want to use the old version of mapred inputformat,  you can use the following parameters like this:`-Dmapred.mapper.new-api=false -Dangel.input.format=org.apache.hadoop.mapred.lib.CombineTextInputFormat`